src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -249,14 +251,36 @@ private void appendRelationship(PrintWriter out, Relationship rel, Reporter repo
         out.print("MATCH ");
         out.print(nodeLookup("n1", rel.getStartNode()));
         out.print(", ");
+        long props = 0;
         out.print(nodeLookup("n2", rel.getEndNode()));
-        out.print(" CREATE (n1)-[:");
+        out.print(" " + typeExport.output() + "(n1)-[n:");
         out.print(quote(rel.getType().name()));
-        long props = formatProperties(out, rel, null);
-        out.println("]->(n2);");
+        formatPropertiesForRelationship(out, rel, typeExport.head(), typeExport.tail());
         reporter.update(0, 1, props);
     }
 
+    private void formatPropertiesForRelationship(PrintWriter out, Relationship rel, String head, String tail){
+        out.print(head);
+        formatProperties(out, rel, null);
+        out.println(tail);
+    }
+
+    private String genericNodeLookup(String id, Node node, TypeExport typeExport){
```

Comment:
not sure why sometimes typeExport is passed as a parameter and sometimes it just uses the field?